tests/playwright/specs/wikipediaHomePage.js

```diff
@@ -1,8 +1,28 @@
+const { saveVideo } = require( 'playwright-video' );
+const path = require( 'path' ),
+	LOGPATH = process.env.LOG_DIR || path.join( __dirname, '..', 'log' );
+
+let capture;
+
 describe( 'Page', () => {
 	beforeAll( async () => {
 		await page.goto( 'https://www.wikipedia.org/' );
 	} );
 
+	beforeEach( async () => {
+		capture = await saveVideo(
+			page,
+			`${LOGPATH}/recording.mp4`
```

Comment:
The order here should be

`beforeEach` -> test -> `afterEach`

this makes it more readable when reading the code sequentially